backend/api/go_http_client/experiment_client/experiment_service/list_experiment_parameters.go

```diff
@@ -126,6 +132,17 @@ func (o *ListExperimentParams) SetHTTPClient(client *http.Client) {
 	o.HTTPClient = client
 }
 
+// WithFilter adds the filter to the list experiment params
+func (o *ListExperimentParams) WithFilter(filter *string) *ListExperimentParams {
+	o.SetFilter(filter)
+	return o
+}
+
+// SetFilter adds the filter to the list experiment params
+func (o *ListExperimentParams) SetFilter(filter *string) {
+	o.Filter = filter
```

Comment:
If this function is so simple, and not widely used, why not just inline it?

Reply:
This is auto-generated by the swagger tool. Everything under api/go_http_client and api/go_client is autogenerated.

Reply:
Ah, sorry, I didn't realize they were auto-generated

Reply:
Sorry, yeah, it's definitely not obvious :-)